tools/run_tests/python_utils/start_port_server.py

```diff
@@ -92,39 +92,37 @@ def start_port_server(port_server_port):
     waits = 0
     while True:
       if waits > 10:
-        print('killing port server due to excessive start up waits')
+        logging.warning('killing port server due to excessive start up waits')
         port_server.kill()
       if port_server.poll() is not None:
-        print('port_server failed to start')
+        logging.error('port_server failed to start')
         # try one final time: maybe another build managed to start one
         time.sleep(1)
         try:
           urllib.request.urlopen('http://localhost:%d/get' % port_server_port,
                           timeout=1).read()
-          print('last ditch attempt to contact port server succeeded')
+          logging.info('last ditch attempt to contact port server succeeded')
           break
         except:
-          traceback.print_exc()
+          logging.exception()
```

Comment:
`TypeError: exception() takes at least 1 argument (0 given)` - what message is worth emitting here?